### ISSUE TITLE
Missing key prop on iterating routes

### ISSUE THREAD

**Issue by davemecha** [Camberi/firecms#105]
When accessing the FireCMS, I get the following warning:

```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `HomeRoute`.
```

**Comment by fgatti675** [comment]
Is that in version 0.43.0?

**Comment by davemecha** [comment]
yes, v 0.43.0

**Comment by fgatti675** [comment]
Should be fixed in v 0.44.0 :)
